TutorialApplication/OrganizationInfoViewController.swift

```diff
@@ -8,10 +8,51 @@
 
 import UIKit
 
+let kEmployeesOrderHasChanged = "faifly.alex.yegorov"
```

Comment:
Find a way to use this constant in Objective-C. Rename its value to something more informative.